README.md

```diff
@@ -795,6 +796,22 @@ iter, err := lo.Attempt(0, func(i int) error {
 // nil
 ```
 
+### Debounce
+creates a debounced instance that delays invoking func given until after wait milliseconds have elapsed.
+```go
+var counter uint64
+f := func() {
+    atomic.AddUint64(&counter, 1)
+}
+debounce := newDebounce(100 * time.Millisecond)
+// in each loop, func given is only invoked once time
+for i := 0; i < 10; i++ {
+    debounce(f, f, f, f, f)
```

Comment:
You chose to set callbacks when calling `debounce()`.

Why did you choose to not set a static list of callbacks from `NewDebounce()` ?

Reply:
what about this:
`d.Add(f2).Add(f2).Add(f2).Add(f2).Add(f2)`